tests/dockerfile/resources/expose_port/fail/Dockerfile

```diff
@@ -4,5 +4,6 @@ COPY package*.json ./
 RUN npm install
 COPY . .
 EXPOSE 3000 22
+EXPOSE 22/tcp
```

Comment:
this is actually not the right place, this check doesn't have a proper test file with examples yet. They need to be added here https://github.com/bridgecrewio/checkov/tree/master/tests/dockerfile/checks

Reply:
@gruebel I have made the code changes, and created the tests. But I actually ran into a bit of an issue, which I ran into last time, but thought it might have just been me not fully understanding the setup.

When I try and setup the dev requirements using `pipenv install --dev` I just get a wall of Red Text.

I believe it's because the `Pipfile.lock` doesn't contain appropriate hashed for my platform `darwin` `arm64`

Should I create a separate issue for this?

Reply:
`pyston` was the main trouble maker, but according to this release note, I'd need to update the `Pipenv` so we can download the `arm64` package

[v2.3.3](https://github.com/pyston/pyston/releases/tag/pyston_2.3.3)

Reply:
@gruebel I could still do with some help on the above, I have the code ready but reluctant to push without testing

Reply:
hey @turingbeing what exactly is your issue at the moment? you can't run the tests or you don't know where to add a test for your change?

Reply:
I've added the tests, but can't run them or install some of the dev-requirements because of architecture.

To make it easier I might just dig out my Linux Laptop and execute them there.

I'll push up my changes meantime but won't request a re-review until I've confirmed the tests pass

Reply:
gotcha, yeah just push your changes and I will check it anyway. it is not such a big change in total 😄